### PR TITLE
fix: Add env.ts to handle environment variables

### PR DESCRIPTION
**Pull request by erdkse**
This PR...

## Changes

- The way of handling environment variables

## Fixes

- NODE_ENV undefined issue

## How to test it

- Copy env.example.ts file as env.ts and set env as production and get the build.

## Screenshots

-

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

**Comment by olensmar** [comment]
@erdkse does this require any changes to the nightly/release build workflows we have on GitHub?

**Comment by erdkse** [comment]
> @erdkse does this require any changes to the nightly/release build workflows we have on GitHub?

Yes, we need to set env variable in env.ts file. Otherwise we can't have build.

**Comment by olensmar** [comment]
> > @erdkse does this require any changes to the nightly/release build workflows we have on GitHub?
> 
> Yes, we need to set env variable in env.ts file. Otherwise we can't have build.

ok thanks - can you make sure this is done for our nightly builds?

**Comment by olensmar** [comment]
sorry for punting on this @erdkse  - i would just like to understand why this is even necessary - since the electron-is-dev solution worked fine in the 1.1.0 release - something must have broken on the way...

**Comment by erdkse** [comment]
Yeah, you are right. `electron-is-dev` library reads ELECTRON_IS_DEV property in `process.env`. But currently, we can't set any value to process.env now. So I made this PR to allow us to continue. If we have time I can investigate what is going on.

**Review by olensmar — APPROVED** [review]

File: electron/main.ts [hunk 1]
```diff
@@ -10,12 +13,18 @@ moduleAlias.addAliases({
   '@root': `${__dirname}/../`,
 });
 
+import ENV from '../env';
+
+if (!ENV.env || ENV.env === 'development') {
```
Comment (olensmar) [inline electron/main.ts:18]: should the logic perhaps be the other way around - i.e. production is default if no file exists? in which case our builds wouldn't have to be updated?

Comment (erdkse) [reply electron/main.ts:18]: Yes, makes sense. I will change in that way.

**Comment by kambydyne** [comment]
I'm merging this in, so we can test it for the release

**Pull request merged**